fe1-web/src/features/evoting/screens/__tests__/CreateElection.test.tsx

```diff
@@ -45,10 +72,97 @@ describe('CreateElection', () => {
 
   it('renders correctly when name and question are empty', () => {
     const component = render(
-      <FeatureContext.Provider value={contextValue}>
-        <MockNavigator component={CreateElection} />
-      </FeatureContext.Provider>,
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
     ).toJSON();
     expect(component).toMatchSnapshot();
   });
+
+  it('renders correctly when name and questions are whitespace string', () => {
+    const { getByTestId, toJSON } = render(
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
+    );
+    const electionName = getByTestId('election_name_selector');
+    const question = getByTestId('question_selector_0_input');
+    const option = getByTestId('question_0_ballots_option_0_input');
+    fireEvent.changeText(electionName, '     ');
+    fireEvent.changeText(question, '        ');
+    fireEvent.changeText(option, '        ');
+    waitFor(() => expect(toJSON()).toMatchSnapshot());
```

Comment:
Not sure this accomplished what you want 😅 Wait for executes the passed function multiple times and succeeds if the passed function succeeds once. (https://testing-library.com/docs/dom-testing-library/api-async/#waitfor)

Reply:
Indeed, it is not needed, but I added it after https://github.com/dedis/popstellar/pull/1456#discussion_r1161118777, because I thought you wanted a more explicit test. My bad.